backends/gaudi/server/text_generation_server/models/flash_causal_lm.py

```diff
@@ -1438,15 +1443,17 @@ def __init__(
         self.kv_cache = []
         self.kv_cache_dtype = dtype if kv_cache_dtype is None else kv_cache_dtype
         self.bucketing_ctx = None
+        htorch.core.hpu_set_env()
         if htorch.utils.internal.is_lazy():
             htorch.hpu.wrap_in_hpu_graph(model, disable_tensor_cache=True)
         environment.set_model_config(self.config)
         self.use_contiguous_pa = (
             os.environ.get("VLLM_CONTIGUOUS_PA", "true").lower() == "true"
         )
-        self.limit_hpu_graphs = (
-            os.environ.get("LIMIT_HPU_GRAPHS", "false").lower() == "true"
+        self.limit_hpu_graph = (
+            os.environ.get("LIMIT_HPU_GRAPH", "false").lower() == "true"
```

Comment:
I don't think we should rename `LIMIT_HPU_GRAPHS` as it may already be used by some users.

Reply:
yes. I just want to keep align with https://github.com/huggingface/text-generation-inference/blob/main/backends/gaudi/server/text_generation_server/models/causal_lm.py#L754, which is "LIMIT_HPU_GRAPH"